assets/src/editor/events/tickets/editor-ticket/use-tickets-for-event-editor-ticket-list.js

```diff
@@ -26,7 +26,10 @@ const useTicketsForEventEditorTicketList = ( isChained ) => {
 
 	const [ finishedResolution, setFinishedResolution ] = useState( false );
 	const { resolveGetRelatedEntitiesForIds } = dispatch( 'eventespresso/core' );
-	if ( filteredDateIds.length && ! finishedResolution ) {
+	if ( ! finishedResolution &&
+		Array.isArray( filteredDateIds ) &&
+		filteredDateIds.length
+	) {
```

Comment:
I didn't realize that `filteredDateIds` could be a non array, thanks 👍

Reply:
i didn't either but then i got an `undefined` error at one point :shrug: